include/StaticAny.h

```diff
@@ -555,7 +555,8 @@ RESTORE_WARNING_STATE
     inline const ElementType* StaticAnyCast(const StaticAny<AnyCastSize,AnyCastAlign>* Any)
     {
         if( std::type_index( typeid(ElementType) ) != std::type_index( Any->get_type() ) ) {
-            throw std::bad_cast();
+            MEZZ_EXCEPTION(BadCastCode, "Invalid StaticAnyCast (const*"
+                                           ").");
```

Comment:
Brain had some issues parsing this as multiple lines.  Seems needless.

Reply:
WTF did I do? Clearly bogus, fixed.